packages/application-extension/src/topbar.ts

```diff
@@ -35,6 +35,7 @@ export const topbar: JupyterFrontEndPlugin<void> = {
   ) => {
     const toolbar = new Toolbar();
     toolbar.id = 'jp-top-bar';
+    toolbar.addAttribute('aria-label', 'Topbar toolbar');
```

Comment:
We need translation wrappers for all user-facing strings, see https://jupyterlab.readthedocs.io/en/stable/extension/internationalization.html#internationalization-and-localization